src/UI/api/utils/queryFilterMapper.test.ts

```diff
@@ -0,0 +1,21 @@
+import { queryFilterMapper } from './queryFilterMapper';
+
+describe(queryFilterMapper.name, () => {
+  it('maps a filter object as expected', () => {
```

Comment:
There's a lot of logic in this method for just one test - are we sure all the options are covered?